app/src/i18n/locales/en/data.json

```diff
@@ -579,11 +578,11 @@
   "fugitive-emissions-oil-gas-type-oil": "Oil",
   "gallons-per-kilometer": "Gallons per kilometer (gal/km)",
   "gallons-per-miles": "Gallons per miles (gal/mi)",
-  "incineration-waste-inboundary-boiler-type": "Boiler type",
+  "boiler-type": "Boiler type",
   "incineration-waste-inboundary-incineration-facility-address": "Incineteration facility address (Optional)",
   "incineration-waste-inboundary-incineration-facility-id": "Incineration facility ID",
-  "incineration-waste-inboundary-technology": "Technology",
-  "incineration-waste-inboundary-waste-composition": "Waste Composition",
+  "technology": "Technology",
+  "waste-composition": "Waste Composition",
```

Comment:
I think we need to keep these like they are unfortunately?
Or why are the strings passed to `t` shortened in this way?

Reply:
in this case the translation strings already exists.